src/scripts/main.js

```diff
@@ -1 +1,90 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  const clickHandler = () => {
+    clearTimeout(timer);
+    document.removeEventListener('click', clickHandler);
+
+    resolve('First promise was resolved');
+  };
+
+  document.addEventListener('click', clickHandler);
+
+  const timer = setTimeout(() => {
+    document.removeEventListener('click', clickHandler);
+
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => {
+    const newElement = document.createElement('div');
+
+    newElement.setAttribute('data-qa', 'notification');
+    newElement.classList.add('success');
+    newElement.innerText = message;
+    document.body.appendChild(newElement);
+  })
+  .catch((error) => {
+    const newElement = document.createElement('div');
+
+    newElement.setAttribute('data-qa', 'notification');
+    newElement.classList.add('error');
+    newElement.innerText = error;
```

Comment:
When setting innerText with an Error object, it's better to use error.message to display the error text. Directly assigning an Error object to innerText will not display a meaningful message.